ibis/backends/duckdb/compiler.py

```diff
@@ -187,14 +187,17 @@ def visit_ArrayZip(self, op, *, arg):
             ]
         )
         func = sge.Lambda(this=body, expressions=[i])
-        return self.f.list_apply(
+        regular = self.f.list_apply(
             self.f.range(
```

Comment:
could instead put the null check up here, so that the input to `list_apply` is NULL. IDK if that is any more performant...

Reply:
I think it's fine as is.